migration-engine/migration-engine-tests/tests/migration_tests.rs

```diff
@@ -623,31 +623,31 @@ async fn changing_a_relation_field_to_a_scalar_field_must_work(api: &TestApi) ->
 #[test_each_connector]
 async fn changing_a_scalar_field_to_a_relation_field_must_work(api: &TestApi) {
     let dm1 = r#"
-            model A {
-                id Int @id
-                b String
-            }
-            model B {
-                id Int @id
-            }
-        "#;
+        model A {
+            id Int @id
+            b String
+        }
+        model B {
+            id Int @id
+        }
+    "#;
     let result = api.infer_and_apply(&dm1).await.sql_schema;
     let table = result.table_bang("A");
     let column = table.column_bang("b");
     assert_eq!(column.tpe.family, ColumnTypeFamily::String);
     assert_eq!(table.foreign_keys, vec![]);
 
     let dm2 = r#"
-            model A {
-                id Int @id
-                b Int
-                b_rel B @relation(fields: [b], references: [id])
-            }
-            model B {
-                id Int @id
-                a A // remove this once the implicit back relation field is implemented
-            }
-        "#;
+        model A {
+            id Int @id
+            b Int
+            b_rel B @relation(fields: [b], references: [id])
+        }
+        model B {
+            id Int @id
+            a A // remove this once the implicit back relation field is implemented
```

Comment:
The comment can be removed as we decided to always enforce both sides of a relation.